clients/docs/includes/configs/cloud/restproxy-sr.config

```diff
@@ -0,0 +1,16 @@
+# Kafka
+bootstrap.servers={{ BROKER_ENDPOINT }}
+security.protocol=SASL_SSL
+sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="{{ CLUSTER_API_KEY }}" password="{{ CLUSTER_API_SECRET }}";
+ssl.endpoint.identification.algorithm=https
```

Comment:
Is `ssl.endpoint.identification.algorithm` required?  The other config files in this folder do not have it

Reply:
The [rest proxy config](https://docs.confluent.io/platform/current/kafka-rest/production-deployment/rest-proxy/config.html) does say default value is `https`, but I copy-pasted it directly from the frontend code for consistency with the previous versions.

@laura-brouckman Do you recall if there was any reason that this was added explicitly and if there would be any implication if I remove this? If I remove it here, the new cloud UI wouldn't have this but is consistency of the config files we give a problem?